### DISCUSSION TITLE
Reload the integration without the button in GUI

### DISCUSSION THREAD

**Discussion by MikeTheCleric** [basnijholt/adaptive-lighting#293] [Q&A]
Hi! Is it possible to reload this integration without use the button in the integration GUI? I need to reload the adaptive lightning after I edit the values from the JSON with an automation or something similar.
Thanks in advance!

**Comment by bemilicus** [comment]
i upvote this one... Clicking on the integration's button (see shot attached) is ok for manual editing.
it would be nice to know how to reload this integration from outside it (ie custom button with call_service or by other means). 
Anyone on this?

TIA!

![shot1](https://user-images.githubusercontent.com/62201229/173026033-0ddd04d0-b0b5-4860-afb4-6898e8622b97.png)

**Comment by MikeTheCleric** [comment]
Anything new about this thread?

**Comment by MikeTheCleric** [comment]
Hi! There's something you can do about this question? I never found a solution for this one. Any help is appreciated.

**Comment by basnijholt** [comment]
If anyone can implement it I will be happy to review and merge it :)
